packages/config/src/Config.types.ts

```diff
@@ -35,6 +35,24 @@ export type ProjectConfig = {
 };
 export type AppJSONConfig = { expo: ExpoConfig; [key: string]: any };
 export type BareAppConfig = { name: string; [key: string]: any };
+export type HookArguments = {
+  config: any;
+  url: any;
+  exp: ExpoConfig;
+  iosBundle: string;
+  iosSourceMap: string | null;
+  iosManifest: any;
+  androidBundle: string;
+  androidSourceMap: string | null;
+  androidManifest: any;
+  projectRoot: string;
+  log: (msg: any) => void;
+};
+export type PostPublishHook = {
+  file: string;
+  config: any;
+  _fn: (input: HookArguments) => any;
```

Comment:
```suggestion
```
I believe `_fn` should not be included here, since it's not a part of the `hooks` field in `ExpoConfig`, it's only added in `Project.ts` when pre-processing the hooks. You could add another type in `Project.ts` for this:
```ts
type LoadedPostPublishHook = PostPublicHook & {
  _fn: (input: HookArguments) => any;
}
```

Reply:
oh good catch, made those changes in the most recent commit 👍